android-o/EmojiCompat/README.md

```diff
@@ -27,7 +27,7 @@ Using this sample requires the Android 8.0 (API 26) and the Xamarin.Android 7.5
 Screenshots
 -------------
 
-<img src="screenshots/Main.png" height="400" alt="Screenshot"/> 
+<img src=“Screenshots/Main.png" height="400" alt="Screenshot"/> 
```

Comment:
markdown image format would be as follows (providing good alt text is essential for accessibility and SEO). don't rely on html image tag and `height` to scale the image, instead upload an image of the correct size 

```
![Description of image for accessibility](Screenshots/Main.png)

```